Makefile.Common

```diff
@@ -0,0 +1,66 @@
+ALL_SRC := $(shell find . -name '*.go' -not -path '*/vendor/*' -type f)
+
+GOTOOLS ?=
+GOTOOLS += " github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.2"
+GOTOOLS += " go.elastic.co/go-licence-detector@v0.5.0"
+GOTOOLS += " github.com/google/addlicense@04bfe4ee9ca5764577b029acc6a1957fd1997153"
```

Comment:
Pinned the latest commit because the latest released version does not include this fix: https://github.com/google/addlicense/pull/125